src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -168,15 +174,22 @@ private bool ValidOptions(ILogger logger)
             }
             else
             {
-                var invalidScopes = AdoPat.Scopes.Validate(this.Scopes);
-                if (!invalidScopes.IsEmpty)
+                if(NoScopeValidation)
+                {
+                    logger.LogWarning($"Received {NoValidationFlag} flag. AzureAuth will skip validation of scopes.");
+                }
+                else
                 {
-                    foreach (var scope in invalidScopes)
+                    var invalidScopes = AdoPat.Scopes.Validate(this.Scopes);
+                    if (!invalidScopes.IsEmpty)
                     {
-                        logger.LogError($"{scope} is not a valid Azure DevOps PAT scope.");
+                        foreach (var scope in invalidScopes)

```

Comment:
The error message does indicate that the scope is not valid and so should look at official Microsoft ADO scope list. However, we should also tell users that if they are still wanting to use that scope, then they can use it with the flag. Basically how will the user discover this flag exists if they really want to use the scope.

Reply:
Apart of running `azureauth ado pat --help` I guess.

Reply:
Added that to the message

Reply:
Thanks @Haard30  for adding. Makes the discoverability easier/better, so better user experience on next steps.